tombstone/types.proto

```diff
@@ -14,7 +14,8 @@ message Tombstone {
   // creator depending on the current NeoFS network settings. A tombstone object
   // must have the same expiration epoch value in `__NEOFS__EXPIRATION_EPOCH`
   // attribute. Otherwise, the tombstone will be rejected by a storage node.
```

Comment:
The behavior should be clarified here, either we don't care about it at all after deprecation, or it must be set in the way described.

Reply:
done